typescript/sdk/src/core/contracts.ts

```diff
@@ -35,16 +38,30 @@ export class CoreContracts extends AbacusAppContracts<CoreContractAddresses> {
     return Inbox__factory.connect(inbox.proxy, this.connection);
   }
 
+  inboxMultisigValidatorManager(
+    chain: ChainName,
+  ): InboxMultisigValidatorManager {
+    const inboxMultisigValidatorManager =
+      this.addresses.inboxMultisigValidatorManagers[chain];
+    if (!inboxMultisigValidatorManager) {
+      throw new Error(`No inboxMultisigValidatorManager for ${chain}`);
+    }
```

Comment:
if it nots a `Partial` we don't need this right?

Reply:
Correct- the reason why it's Partial is that all chains, including testnets / test chains, are included in the `ChainName` type :/
```
export type ChainName = MainnetChainNames | TestnetChainNames | TestChainNames;
```